### PR TITLE
Implemented TreeGrid tables inspired by bootstrap-treeview

### PR DESCRIPTION
**Pull request by skateman**
screenshot from 2016-01-28 16-09-05

### CODE REVIEW

File: less/tables.less [hunk 1]
```diff
@@ -68,3 +68,20 @@
...
+  span.indent {
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+  span.icon {
+    margin-right: 5px;
+    display: inline-block;
+    font-size: 13px;
+    min-width: 10px;
+    text-align: center;
```
Comment (jeff-phillips-18) [inline less/tables.less:82]: Declarations should be ordered alphabetically by property.

**Comment by jeff-phillips-18** [comment]
Please include the generated files with this PR. Files such has the generated css files and the tests files.

**Comment by skateman** [comment]
@jeff-phillips-18 should be good now

**Comment by jeff-phillips-18** [comment]
I'm not sure why the tests.css include was removed from all the generated tests pages.

**Comment by skateman** [comment]
It was weird for me too, I just typed `grunt` into the command line. Is that okay for building?

**Comment by jeff-phillips-18** [comment]
It should be. Maybe your jekyll version is not right? When you build (or just grunt) you should see:

Running "jekyll:tests" (jekyll) task
`jekyll build --source tests-src --destination tests` was initiated.

Configuration file: tests-src/_config.yml
            Source: tests-src
       Destination: tests
      Generating... 

The inclusion of the tests.css is in the _config.yml file.

I have jekyll 2.5.3

**Comment by skateman** [comment]
jekyll 3.1.0 here, should I downgrade?

**Comment by skateman** [comment]
Same happens with 2.5.3 :(

**Comment by jeff-phillips-18** [comment]
When I upgraded to 3.1.1, I get the same issue. However, I cannot seem to go back to 2.5.3 now. How did you downgrade?

**Comment by jeff-phillips-18** [comment]
Sorry. I uninstalled 3.1.1 and got back to 2.5.3 and it works fine now. So, it must be a version problem. What does your jekyll --version say now?

**Comment by skateman** [comment]
should be good now!

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by skateman** [comment]
yay :+1:

**Comment by jeff-phillips-18** [comment]
@skateman Please squash the commits.

**Comment by skateman** [comment]
@jeff-phillips-18 done

**Comment by skateman** [comment]
@epwinchell FYI

**Comment by epwinchell** [comment]
:cake:

**Pull request merged**